include/swift/Runtime/Backtrace.h

```diff
@@ -50,7 +57,11 @@ typedef int ErrorCode;
 
 SWIFT_RUNTIME_STDLIB_INTERNAL ErrorCode _swift_installCrashHandler();
 
+#ifdef __linux__
+SWIFT_RUNTIME_STDLIB_INTERNAL bool _swift_spawnBacktracer(const ArgChar * const *argv, int memserver_fd);
```

Comment:
Moving my feedback from the other PR: would this benefit from a doc comment that clarifies the use of each parameter and the return pointer as well? Even if it's already in the `.cpp` file, IUUC editors with support for displaying documentation on hovered symbol load it from `.h` files.